.github/workflows/charts.yml

```diff
@@ -0,0 +1,64 @@
+name: Charts
+
+on:
+  pull_request:
+    branches: [ main, release/** ]
+  push:
+    branches: [ main, release/** ]
+    tags: [ v* ]
```

Comment:
If there is a path specified here it will only run when there are changes to the `helm-chart` directory where the chart configurations all reside.
```
push:
  path:
    - helm-chart/**
```

Reply:
Actually I'd want it to run all the time to see that some product or config changes don't break the chart deployment so we find out in advance